src/gui/mzroll/loginform.cpp

```diff
@@ -57,40 +58,39 @@ void LoginForm::on_pushButton_clicked()
     workerThread->start();
 }
 
-
-void LoginForm::handleResults(QStringList results){
-    QString status_inside=results.at(0);
-    QString username=results.at(1);
-    QString password=results.at(2);
-    if (status_inside=="ok"){
-        qDebug()<<"Logged in, moving on now....";
+void LoginForm::handleResults(QStringList results)
+{
+    QString status_inside = results.at(0);
+    QString username = results.at(1);
+    QString password = results.at(2);
+    if (status_inside == "ok") {
+        qDebug() << "Logged in, moving on now....";
         ui->login_label->setText("Fetching data from Polly..");
         QCoreApplication::processEvents();
         _pollyelmaveninterfacedialog->credentials = QStringList()<< username << password;
```

Comment:
@shubhra-agrawal 
We don't need to store credentials in this pointer now. Initially we were storing these to achieve persistent login in one session but now we have persistent login functionality till the users explicitly click on "logout" button.
Please remove this line as it imposes security risk of exposing user credentials.